cves/CVE-2016-2327.yml

```diff
@@ -64,7 +64,9 @@ description_instructions: |
 
   Your target audience is people just like you before you took any course in
   security
-description:
+description: |
+  An out-of-bounds array access could be triggered by a user crafting a custom .avi file that would cause an error
+  in performing row calculations.
```

Comment:
Could use some more here. Remember that you audience may not know what an AVI file is, and how one could actually craft it. What part of the format was broken? What's a row calculation? Take look at the "great work" pull requests to see what level of detail we're expecting.